cnxdb/migrations/20170911201035_transform_cnxml_to_html.py

```diff
@@ -53,11 +74,12 @@ def up(cursor):
     """Transform all index.cnxml to index.cnxml.html"""
     # Get all the index.cnxml.html that does not have rhaptos.cnxmlutils
     # current version
-    to_transform = should_run(cursor, limit='')
-    num_todo = len(to_transform)
+    raw_to_transform = should_run(cursor, raw=True, baked=False, limit='')
+    baked_to_transform = should_run(cursor, raw=False, baked=True, limit='')
```

Comment:
These kinda make me wonder if there shouldn't be two different functions that share a base function.

Reply:
You're absolutely right... :joy:

Reply:
I've created 2 functions for this so it should be clearer now :sweat_smile: